tests/generic_config_updater/test_pg_headroom_update.py

```diff
@@ -79,7 +79,7 @@ def _confirm_value_in_app_and_asic_db():
 
 @pytest.mark.parametrize("operation", ["replace"])
 def test_pg_headroom_update(duthost, ensure_dut_readiness, operation, skip_when_buffer_is_dynamic_model):
-    asic_type = duthost.get_asic_name()
+    asic_type = get_asic_name(duthost)
```

Comment:
`sonic-cfggen -d -v DEVICE_METADATA.localhost.hwsku` on vs seems to be recognized as `Force10-S6000` which leads to `td2`. May be we can add a comment?